packages/contract-tests/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@odh-dashboard/contract-tests",
+  "version": "1.0.0",
+  "description": "Zero-config contract testing for ODH Dashboard packages",
+  "scripts": {
+    "type-check": "tsc -p tsconfig.json --noEmit",
```

Simplify to be consistent with other packages.

```suggestion
    "type-check": "tsc --noEmit",
```